spec/spec_helper.rb

```diff
@@ -27,4 +29,5 @@
   c.default_facts = default_facts
 end
 
+add_custom_fact :pe_version, '2016.4.2'
```

Comment:
Remedied, testing is processing slowly of course. Editing from an iPad so can't rebase, the commits will need squashed during merge.